journal/original/17-chapter-14.adoc

```diff
@@ -433,6 +441,11 @@ and all that wrong us, we leave to the Lord.
 This is to clear our innocency from the aspersion cast upon us,
 that we are seditious or plotters.`"
 
+--
+
+[.embedded-content-document.letter]
+--
+
 [.blurb]
```

Comment:
I'm not sure if this should be separated from the letter above.

Reply:
Yeah, I think I would have done what you did here. Kind of a tough call, but I agree.